CorsixTH/Lua/rooms/ward.lua

```diff
@@ -135,14 +153,20 @@ function WardRoom:commandEnteringPatient(patient)
     print("Warning: A patient was called into the ward even though there are no free beds.")
   else
```

Comment:
Instead of a long `else`, deal with the `if` case completely, and return from the function.
Code is simpler if it doesn't continue executing stuff 25 lines down which you can only find by finding the correspond `end`.

Also, the `else` can be dropped and the entire `else` case can be dedented 2 spaces.